### PR TITLE
docs(pattern): creation flows

### PR DESCRIPTION
**Pull request by connor-leech**
Migrated the Creation flows pattern to the new site repo.

https://github.com/carbon-design-system/carbon-website-archive/pull/1655

### CODE REVIEW

**Comment by laurenmrice** [comment]
didnt realize you didnt want a review and just are trying to merge over ! :P will approve

**Comment by connor-leech** [comment]
@laurenmrice thanks for the thorough feedback! I haven't actually done a copy pass on this yet. I believe @jeanservaas is going to clean up these images. I'll re-request your review once the copy has been reworked.

File: src/data/nav-items.yaml [hunk 1]
```diff
@@ -126,6 +126,8 @@
       path: /experimental/overview
     - title: Chatbots
       path: /experimental/chatbot/overview
+    - title: Creation flows
```
Comment (janchild) [inline src/data/nav-items.yaml]:
```suggestion
    - title: Create flows
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 2]
```diff
@@ -0,0 +1,267 @@
+---
+title: Creation flows
+---
+
+<PageDescription>
+
+Creation is the generation of a new resource. This pattern is distinct from adding, which is the movement of an existing resource.
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
To create is to generate a new resource. Creating is distinct from adding, but the two are often confused. To add a resource is to include a resource that already exists.
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+![Example of a full-page creation flow](images/5.png)
+
+<Caption>Example of a full-page creation flow</Caption>
+
+</Column>
+</Row>
+
+## Multistep creation
+
+If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependant options together.
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependent options together.
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+Multistep creation follows the same principles as standard creation but the creation takes place through several steps. This flow can take place in a side panel or can use the full display.
+
+### Side panel
+
+<Row>
+<Column colLg={8}>
+
+![Example of a side panel multi-step creation flow](images/11.png)
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
![Example of side panels in a multi-step creation flow](images/11.png)
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+</Column>
+</Row>
+
+## Multistep creation
+
+If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependant options together.
+
+Multistep creation follows the same principles as standard creation but the creation takes place through several steps. This flow can take place in a side panel or can use the full display.
+
+### Side panel
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
### Side panel multistep
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+<Column colLg={8}>
+
+![ExampExample of a side panel multi-step creation flow](images/12.png)
+
+<Caption>Example of a side panel multi-step creation flow</Caption>
+
+</Column>
+</Row>
+
+### Full page
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
### Full page multistep
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+<Column colLg={8}>
+
+![Multistep creation flow with image](images/15.png)
+
+<Caption>Multistep creation flow with image</Caption>
+
+</Column>
+</Row>
+
+### Side panel
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion

```

File: src/pages/experimental/creation-flows/index.mdx [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+![Example of a multi-step creation flow](images/11.png)
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+
+<Caption>Example of a multi-step creation flow.</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+
+![Example of a multi-step creation flow.](images/12.png)
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+
+<Caption>Example of a multi-step creation flow..</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 12]
```diff
@@ -0,0 +1,267 @@
...
+
+### Full page
+
+![Multistep creation flow with image](images/13.png)
+
+<Caption>Multistep creation flow with image.</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
<Caption>Multistep creation flow with image</Caption>
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 13]
```diff
@@ -0,0 +1,267 @@
...
+
+![Multistep creation flow with image](images/14.png)
+
+<Caption>Multistep creation flow with image.</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
<Caption>Multistep creation flow with image</Caption>
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 14]
```diff
@@ -0,0 +1,267 @@
...
+
+![Multistep creation flow with image](images/15.png)
+
+<Caption>Multistep creation flow with image.</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
<Caption>Multistep creation flow with image</Caption>
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 15]
```diff
@@ -0,0 +1,267 @@
...
+
+### Confirmation
+
+When any create action is a success, a notification banner can be presented to visually confirm this.
+
+![Example of a successful creation confirmation"](images/16.png)
+
+<Caption>Example of a successful creation confirmation.</Caption>
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
<Caption>Example of a successful creation confirmation</Caption>
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 16]
```diff
@@ -0,0 +1,267 @@
...
+
+![Example of a multi-step creation flow](images/11.png)
+
+<Caption>Example of a multi-step creation flow.</Caption>
+
+![Example of a multi-step creation flow.](images/12.png)
+
+<Caption>Example of a multi-step creation flow..</Caption>
+
+### Full page
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
### Full page multistep with image
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 17]
```diff
@@ -0,0 +1,267 @@
...
+
+![Example of a successful creation confirmation"](images/16.png)
+
+<Caption>Example of a successful creation confirmation.</Caption>
+
+## Anatomy
+
+### Button position
+
+When the full-page dialog is viewed in mobile, the buttons are positioned at the bottom of the screen.
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
When the full-page modal is viewed in mobile, the buttons are positioned at the bottom of the screen.
```

Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
When the full-page is viewed in mobile, the buttons are positioned at the bottom of the screen.
```

Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
When the full page is viewed in mobile, the buttons are positioned at the bottom of the screen.
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 18]
```diff
@@ -0,0 +1,267 @@
...
+
+![If an illustration or image is displayed on the right then buttons should be displayed above the image.](/images/7.9.png)
+
+</DoDontExample>
+</Column>
+</Row>
+
+#### Including an image
+
+To include a supporting image, place it on the right-hand side of the window.
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
When including a supporting image, place it on the right-hand side of the window.
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 19]
```diff
@@ -0,0 +1,262 @@
...
+
+![Example of a successful creation confirmation"](images/16.png)
+
+<Caption>Example of a successful creation confirmation</Caption>
+
+## Anatomy
+
+### Button position
+
+When the full page is viewed in mobile, the buttons are positioned at the bottom of the screen.
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
When viewed on mobile, the buttons are positioned at the bottom of the screen.
```

File: src/pages/experimental/creation-flows/index.mdx [hunk 20]
```diff
@@ -0,0 +1,262 @@
+---
+title: Creation flows
```
Comment (janchild) [inline src/pages/experimental/creation-flows/index.mdx]:
```suggestion
title: Create flows
```

**Pull request merged**